### PR TITLE
Doc 8135 travel sample

### PR DESCRIPTION
**Pull request by osfameron**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Set the base image to Node
+FROM node
+
+# File Author / Maintainer
+MAINTAINER Couchbase
```
Comment (maria-robobug) [inline Dockerfile]: We can change this to `LABEL`: https://docs.docker.com/engine/reference/builder/#maintainer-deprecated

So you could do: `LABEL maintainer=Couchbase`

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+# Expose ports
+EXPOSE 8081
+
+COPY . .
+
+# Set the default command to execute
+CMD npm run serve
+
+# try:
```
Comment (maria-robobug) [inline Dockerfile]: Probably we would have this in the Readme no? Not sure if necessary here

File: README.md [hunk 3]
```diff
@@ -1,29 +1,49 @@
...
+
+## How to run
+
+### Quickstart: via backend
+
+We recommend running from the perspective of one of the backends: this will build the backend code,
+couchbase database, and the frontend.
+The following backends are currently available, we intend to provide one for each SDK:
+
+ * [Python](https://github.com/maria-robobug/try-cb-python/tree/DOC-8136)
```
Comment (maria-robobug) [inline README.md]: 📆 (reminder) - we should change this reference to  the couchbaselabs url

File: README.md [hunk 4]
```diff
@@ -1,29 +1,49 @@
...
+    docker-compose up backend db
+
+    Run the frontend server locally. This will auto-restart when you modify the code.
+    npm run serve
+
+If you prefer not to use docker, you can run the backend and/or couchbase server separately.
+One useful case is running the db locally, and passing its IP address directly to the backend:
+
+    # example only, use the IP address of your locally running db server
+    ARGS="-c 10.144.211.101" docker-compose up backend
```
Comment (maria-robobug) [inline README.md]: This has changed a bit, I've split it into 3 env variables: 
CB_HOST
CB_USER
CB_PSWD

So you could say:
`CB_HOST=10.144.211.101 docker-compose up backend`

File: docker-compose.yml [hunk 5]
```diff
@@ -0,0 +1,34 @@
+version: "3.9"
+
+services:
+  backend:
+    build: "https://github.com/maria-robobug/try-cb-python.git#DOC-8136"
```
Comment (maria-robobug) [inline docker-compose.yml]: 📆 (reminder) - Replace with couchbaselabs url

File: docker-compose.yml [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+    depends_on:
+      db:
+        condition: service_healthy
+    ports:
+      - 8080:8080
+    environment:
+      - ARGS
+    command: $ARGS
```
Comment (maria-robobug) [inline docker-compose.yml]: You will need to update `backend` and `db` definitions a bit, so we're in sync:

https://github.com/couchbaselabs/try-cb-python/pull/20/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3

Obviously ignore the `volume` part for the backend, as you won't need it in this project.

File: src/components/Basket.vue [hunk 7]
```diff
@@ -37,10 +37,10 @@
 
 <script>
 import axios from 'axios'
-import {config} from '../main.js'
 
 export default {
   name: "basket",
+  inject: ['API'],
```
Comment (maria-robobug) [inline src/components/Basket.vue:42]: 👍 nice!

File: src/App.vue [hunk 8]
```diff
@@ -41,8 +42,16 @@
   import Basket  from './components/Basket.vue'
   import Booked  from './components/Booked.vue'
   import Hotels  from './components/Hotels.vue'
-  export default {
+
+export default {
```
Comment (osfameron) [inline src/App.vue]: bad indentation

File: src/components/Booked.vue [hunk 9]
```diff
@@ -22,10 +22,11 @@
 
 <script>
 import axios from 'axios'
-import {config} from '../main.js'
+import {config, tenantedAPI} from '../main.js'
```
Comment (osfameron) [inline src/components/Booked.vue]: redundant

File: public/index.html [hunk 10]
```diff
@@ -12,7 +12,10 @@
     <noscript>
       <strong>We're sorry but try-cb-frontend-v2 doesn't work properly without JavaScript enabled. Please enable it to continue.</strong>
     </noscript>
-    <div id="app"></div>
+
+    <div id="app">
+    </div>
```
Comment (maria-robobug) [inline public/index.html:17]: Do we need the closing `</div>` tag in a newline, or is this a formatting thing maybe?

**Review by maria-robobug — COMMENTED** [review]
LGTM! 👍 just added a few comments!

File: public/index.html (hunk 10 above)
Comment (osfameron) [reply public/index.html:17]: Just an editing artefact, (e.g. I added something, then removed it), sorry :)

**Comment by maria-robobug** [comment]
LGTM! Nice work on the `wait-for-it` addition! 👍

**Comment by osfameron** [comment]
closing and reissuing against 7.0

**Pull request closed without merging**